### PR TITLE
New optional attribute for nexxtv-player extension

### PR DESCRIPTION
**Pull request by neko-fire**
Additional attribute (data-disable-ads) to disable ads for nexxtv player.

### CODE REVIEW

**Comment by neko-fire** [comment]
I signed it!

File: extensions/amp-nexxtv-player/0.1/amp-nexxtv-player.js [hunk 1]
```diff
@@ -106,6 +107,10 @@ class AmpNexxtvPlayer extends AMP.BaseElement {
     src += `?start=${encodeURIComponent(start)}`;
     src += `&datamode=${encodeURIComponent(mode)}&amp=1`;
 
+    if (disableAds === '1') {
+      src += `&disableAds=${encodeURIComponent(disableAds)}`;
```
Comment (aghassemi) [inline extensions/amp-nexxtv-player/0.1/amp-nexxtv-player.js]: can just be  `+= '&disableAds=1'`

**Comment by aghassemi** [comment]
closing/reopening for CLA check to restart

**Comment by aghassemi** [comment]
@neko-fire, CLA check is stuck on this PR, do you mind submitting another PR and closing this one? Thanks!

**Pull request closed without merging**